application.py

```diff
@@ -2,8 +2,15 @@
 
 import os
 from app import create_app
+from app import settings
+
 from flask.ext.script import Manager, Server
 
+if not settings.EQ_PRODUCTION:
+    with open(os.getcwd() + '/jwt-test-keys/rrm-public.pem', "r") as public_key_file:
```

Comment:
Remove and keep only one method of loading keys (i.e. set the env value) ~ do it outside prior to running locally